plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -912,12 +912,27 @@ public TableStatistics getTableStatistics(ConnectorSession session, JdbcTableHan
     @Override
     public void createSchema(ConnectorSession session, String schemaName)
     {
+        try (Connection connection = connectionFactory.openConnection(session)) {
+            if (connection.getMetaData().storesUpperCaseIdentifiers()) {
+                schemaName = schemaName.toUpperCase(ENGLISH);
```

Comment:
Can you test if after this change a query runner with `case-insensitive-name-matching` disabled is able to work at all?
I think it would fail consistently.

Reply:
Oracle doesn't implement `createSchema`. Isn't `case-insensitive-name-matching` disabled by default?